### PR TITLE
Fixes needed to be able to compile pdfminer.six with Cython

### PR DESCRIPTION
**Pull request by mawoqiw**
Per comment in the original project https://github.com/euske/pdfminer/issues/140 compiling PDFMiner with Cython leads to substantial speed improvements. Submitted fixes are necessary to be able to compile pdfminer.six,  otherwise compiler fails.

### CODE REVIEW

**Comment by timb07** [comment]
I'm quite interested in this. I've made some other improvements to speed up processing for certain inputs (#133, #141), but my next target for speeding up is `psparser.py`, and I haven't found any obvious improvements to make there. However, compiling just that file with Cython does give a small speed increase.

File: pdfminer/ccitt.py [hunk 1]
```diff
@@ -15,6 +15,8 @@
 
 import six  #Python 2+3 compatibility
 
+import unittest
```
Comment (timb07) [inline pdfminer/ccitt.py]: What is this needed for? Was it included by mistake?

File: setup.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 import pdfminer as package
 
-requires = ['six', 'pycryptodome']
+requires = ['six', 'pycryptodome', 'pygame']
```
Comment (timb07) [inline setup.py]: Again, what is `pygame` needed for?

Comment (mawoqiw) [reply setup.py]: A function from `unitest` is called [here](https://github.com/mawoqiw/pdfminer.six/blob/4bdb3ba8cc79259336f797d00d9a48e93c53bdba/pdfminer/ccitt.py#L581) without an import.
`pygame` is not listed among requirements but is called [here](https://github.com/mawoqiw/pdfminer.six/blob/4bdb3ba8cc79259336f797d00d9a48e93c53bdba/pdfminer/ccitt.py#L585) and [here](https://github.com/mawoqiw/pdfminer.six/blob/4bdb3ba8cc79259336f797d00d9a48e93c53bdba/pdfminer/ccitt.py#L599)

So far I haven't found much space for improvement in `psparser.py,  other than this, either.

**Comment by mawoqiw** [comment]
One of the tests is using `image.save` method from that library. I guess it could be replaced by something else to get rid of that dependency.

File: setup.py (hunk 2 above)
Comment (pombredanne) [reply setup.py]: @mawoqiw I think we should not add pygame as a dep... instead it should be removed. When was this introduced?

File: pdfminer/ccitt.py (hunk 1 above)
Comment (pombredanne) [reply pdfminer/ccitt.py]: This should be moved in the `main()` and/or the main removed entirely and moved to some testing script outside of the main library code

File: setup.py (hunk 2 above)
Comment (pombredanne) [reply setup.py]: This  `main()` should be removed entirely and moved to some testing script outside of the main library code  and we cannot add pygame as a runtime dep

Comment (mawoqiw) [reply setup.py]: You are right @pombredanne. `pygame` is indeed not required to use Cython, thanks. 
With regards to `main()` function I would rather deal with it in a different merge request. Do you have any idea what is the purpose of that code?

**Pull request merged**